financial-templates-lib/clients/ExpiringMultiPartyClient.js

```diff
@@ -157,26 +189,19 @@ class ExpiringMultiPartyClient {
     this.expiredLiquidations = expiredLiquidations;
     this.disputedLiquidations = disputedLiquidations;
 
```

Comment:
I changed the below from the reduce to a map as there is no longer zero collateral positions due to the filtering above and the map is way easier to read.